template/tsconfig/vitest/package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "scripts": {
+    "typecheck": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false"
```

Comment:
This would overwrite the one in the typescript template: https://github.com/vuejs/create-vue/blob/d8723fb59c34d3e5599442903170240417efb47c/template/config/typescript/package.json#L4 and only type-check the specs.
I think we need a separate script or merge it with the default `typecheck` script or the `test:unit` script

Reply:
What about `    "typecheck": "vue-tsc --noEmit && vue-tsc --noEmit -p tsconfig.vitest.json --composite false"` then?

Reply:
Looks good to me 👍

Reply:
PR updated!